aries-site/src/examples/templates/dashboards/DashboardExample.js

```diff
@@ -1,6 +1,210 @@
-import React from 'react';
-import { DashboardTiles } from '.';
-import { AppHeaderExample, PageHeaderExample, ScreenContainer } from '..';
+import React, { useContext } from 'react';
+import { AppIdentity, Identifier, Tiles, Tile } from 'aries-core';
+import {
+  Avatar,
+  Header,
+  Heading,
+  Button,
+  Box,
+  Menu,
+  ResponsiveContext,
+  Footer,
+  Text,
+} from 'grommet';
+import PropTypes from 'prop-types';
+
+import {
+  Apps,
+  Chat,
+  Clock,
+  Terminal,
+  StatusUnknown,
+  Wifi,
+  System,
+  User,
+  Tasks,
+  Location,
+  ShieldSecurity,
+} from 'grommet-icons';
+
+const data = [
+  {
+    color: 'blue',
+    icon: <Wifi />,
+    title: 'Remote Access',
+    subTitle: 'Lights out Management (LOM)',
+    message: 'Connected',
+  },
+  {
+    color: 'green',
+    icon: <System />,
+    title: 'System',
+    subTitle: 'Sub-system and Devices',
+    message: 'Composable System',
+  },
+  {
+    color: 'red',
+    icon: <User />,
+    title: 'User Sessions',
+    subTitle: 'User Access on Server',
+    message: '4 active sessions',
+  },
+  {
+    color: 'purple',
+    icon: <Tasks />,
+    title: 'Logs',
+    subTitle: 'Events, Integration, and Status',
+    message: '204,353',
+  },
+  {
+    color: 'orange',
+    icon: <Location />,
+    title: 'Beacons',
+    subTitle: 'Uniqe identification light',
+    message: '24 beacons connected',
+  },
+  {
+    color: 'teal',
+    icon: <ShieldSecurity />,
+    title: 'Security',
+    subTitle: 'Trusted Platform Module',
+    message: 'No Module Connected',
+  },
+];
+
+const DashboardTiles = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Tiles
+      gap="medium"
+      columns={size !== 'small' ? 'small' : 'auto'}
+      margin={size === 'small' ? { bottom: 'xlarge' } : undefined}
+    >
+      {data.map(value => (
+        <Tile
+          background={value.color}
+          key={`Tile ${value.title}`}
+          alignContent="center"
+          onClick={() => {}}
+        >
+          <Identifier
+            pad={{ horizontal: 'medium', vertical: 'small' }}
+            title={value.title}
+            subTitle={value.subTitle}
+            size="small"
+            gap="medium"
+            direction="column"
+            align="start"
+          >
+            {value.icon}
+          </Identifier>
+          <Box flex />
+          <Footer pad="small" background="background-contrast">
+            <Text size="xsmall">{value.message}</Text>
+            {value.message === 'Connected' && (
+              <Box round pad="xsmall" background="green" />
+            )}
+          </Footer>
+        </Tile>
+      ))}
+    </Tiles>
+  );
+};
+
+const SidebarExample = ({ ...rest }) => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box
```

Comment:
This should probably be updated to use Sidebar

Reply:
done good point!